src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/terminal/terminal.utils.tsx

```diff
@@ -0,0 +1,210 @@
+import ReactSelect from 'react-select'
+import { TippyCustomized, TippyTheme } from '@devtron-labs/devtron-fe-common-lib'
+import {
+    SelectWrapperType,
+    ReactSelectType,
+    WrapperTitleType,
+    ConnectionButtonType,
+    CloseExpandView,
+    ConnectionSwitchType,
+    ClearTerminalType,
+} from './terminal.type'
+
+const creatableSelectWrapper = (selectData: SelectWrapperType) => {
+    if (selectData.hideTerminalStripComponent) return null
+    return (
+        <>
+            <span className="bcn-2 mr-8" style={{ width: '1px', height: '16px' }} />
+            <div className="cn-6 ml-8 mr-4">{selectData.title}</div>
+            {selectData.showInfoTippy && (
+                <TippyCustomized
+                    theme={TippyTheme.white}
+                    heading="Image"
+                    placement="top"
+                    interactive={true}
+                    trigger="click"
+                    className="w-300"
+                    Icon={Help}
+                    showCloseButton={true}
+                    iconClass="icon-dim-20 fcv-5"
+                    additionalContent={selectData.infoContent}
+                >
+                    <HelpIcon className="icon-dim-16 mr-8 cursor" />
+                </TippyCustomized>
+            )}
+            <div>
+                <CreatableSelect
+                    placeholder={selectData.placeholder}
+                    options={selectData.options}
+                    defaultValue={selectData.defaultValue}
+                    value={selectData.value}
+                    onChange={selectData.onChange}
+                    styles={selectData.styles}
+                    components={selectData.components}
+                />
+            </div>
+        </>
+    )
+}
+
+const reactSelect = (selectData: ReactSelectType) => {
+    if (selectData.hideTerminalStripComponent) return null
+    return (
+        <>
+            {selectData.showDivider && <span className="bcn-2 mr-8" style={{ width: '1px', height: '16px' }} />}
+            <div className="cn-6 mr-10">{selectData.title}</div>
+            <div style={{ minWidth: '145px' }}>
+                <ReactSelect
+                    placeholder={selectData.placeholder}
+                    options={selectData.options}
+                    defaultValue={selectData.defaultValue}
+                    value={selectData.value}
+                    onChange={selectData.onChange}
+                    styles={selectData.styles}
+                    components={selectData.components}
+                />
+            </div>
+        </>
+    )
+}
+
+const titleName = (titleData: WrapperTitleType) => {
+    if (titleData.hideTerminalStripComponent) return null
+    return (
+        <>
+            <div className="cn-6 mr-16">{titleData.title}</div>
+            <div className="flex fw-6 fs-13 mr-20">{titleData.value}</div>
+            <span className="bcn-2 mr-16 h-32" style={{ width: '1px' }} />
```

Comment:
Can we avoid this by using right border?

Reply:
leaving it the same for now due to height of divider differs from parent component height

Reply:
Can use css formula for width bw-1